src/NUnitTestAdapterTests/TestExecutionTests.cs

```diff
@@ -153,9 +158,48 @@ public void TestResultIsReportedCorrectly(string name, TestOutcome outcome, stri
         }
 
         [Test]
-        public void ExplicitTestDoesNotShowUpInResults()
```

Comment:
yes it is remove (replaced with the additional Testcase for TestResultIsReportedCorrectly) 

The old test was flawed (it worked only because testResults where never initialized and always empty)